### ISSUE TITLE
Improve handling lines with specific prefixes

### ISSUE THREAD

**Issue by Okeanos** [adrienverge/yamllint#687]
When using the `# yaml-language-server: $schema=<URL>` syntax to declare a YAML schema inline (as described [here](https://github.com/redhat-developer/yaml-language-server/blob/main/README.md#using-inlined-schema)) the resulting line is of arbitrary length.

This obviously conflicts with settings such as 

```yaml
rule:
  line-length:
    max: 140
```

Even setting `allow-non-breakable-inline-mappings: true` doesn't help here. I can of course set custom disables and write it like this:

```yaml
# yamllint disable-line rule:line-length
# yaml-language-server: $schema=https://raw.githubusercontent.com/asciidoctor/asciidoctor-intellij-plugin/main/src/main/resources/jsonSchemas/antoraPlaybookSchema.json

```

However, that doesn't look quite nice and is error-prone.

A way of globally specifying to ignore lines starting with a particular prefix doesn't appear to exist. Would it be possible to add that or is there an altogether different way of handling this that I am overlooking?

**Comment by Okeanos** [comment]
This may actually be a duplicate of https://github.com/adrienverge/yamllint/issues/545 unless there's another proposal for my question … searching is hard :/

**Comment by adrienverge** [comment]
Hello @Okeanos, thank you for the proposal.

> A way of globally specifying to ignore lines starting with a particular prefix doesn't appear to exist. Would it be possible to add that or is there an altogether different way of handling this that I am overlooking?

Indeed such config option doesn't exist yet. But as you've found out, it's already proposed in #545, so we can continue there.

**Comment by Okeanos** [comment]
Thanks for confirming. I'll close this as duplicate then.

**Comment by andrewimeson** [comment]
Specific to the YAML schema headers, I've found that removing the space still works:

```yaml
# yaml-language-server:$schema=https://raw.githubusercontent.com/asciidoctor/asciidoctor-intellij-plugin/main/src/main/resources/jsonSchemas/antoraPlaybookSchema.json
```
